reactioncommerce/reaction-next-starterkit
# Add "Order Completed" event tracking

Resolves #380 
Impact: minor
Type: feature

NOTE: Reaction needs to be on the RC5 branch and the blocking issues needs to have been merged.

### Summary
Adds tracking for the "Order Completed" event

## Testing
1. Place an order
2. Verify that an "Order Completed" event was published to Segment
